src/styles/index.scss

```diff
@@ -1,3 +1,90 @@
-body {
-  margin: 0;
+@import "./utils/variables";
+@import "./utils/normalize";
+@import "./utils/reset";
+@import "./star.scss";
+
+.product-card {
```

Comment:
These classes should be moved to a separate file in order to divide the styling into BEM blocks.